src/main/java/com/google/api/generator/spring/composer/SpringAutoConfigClassComposer.java

```diff
@@ -888,7 +876,7 @@ private static MethodDefinition createClientBeanMethod(
                 .build());
 
     String methodName =
-        CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_CAMEL, service.name()) + "Client";
+        CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_CAMEL, service.name()) + "Settings";
```

Comment:
nit: perhaps extract this `methodName` that's also used in `createClientBeanMethod()` for consistency between the two beans.

Reply:
Now using a common variable in the facade method